cmd/deploy/deploy.go

```diff
@@ -270,6 +270,39 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 			deployOptions.Manifest.Deploy.ComposeSection.Stack.Name = deployOptions.Name
 
 			mergeServicesToDeployFromOptionsAndManifest(deployOptions)
+			if len(deployOptions.servicesToDeploy) > 0 {
```

Comment:
could we have a unit test for this logic?

Reply:
Yes! I was planning to do it